src/map.c

```diff
@@ -131,6 +132,11 @@ void map_cleanup(void)
  */
 gboolean map_keypress(GtkWidget *widget, GdkEventKey* event, gpointer data)
 {
+    if (is_processing_events()) {
+        // events are processing, pass all keys unmodified
+        return false;
```

Comment:
This is only for when we send the events to GDK/GTK - see process_events() function